examples/offsetAnchorPosition.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<script src="../third_party/web-animations-js/web-animations.dev.js"></script>
+<script src="../index.min.js"></script>
+
+<div id="container">
+  <div id="target">
+  </div>
+</div>
+
+<style>
+  body {
+    margin: 0px;
+  }
+
+  #container {
+
+    position: absolute;
+    left: 300px;
+    top: 300px;
+    background-color: yellow;
+
+    width: 1000px;
+    height: 1000px;
```

Comment:
I don't think we should assume the user's viewport can contain a 1000px 1000px rect.
I'm not particularly happy with the quality of these demos but it's beyond the scope of this patch, feel free to ignore for now, we can focus on demo quality later.